pycbc/events/coinc.py

```diff
@@ -1010,6 +1010,10 @@ def _find_coincs(self, results, ifos):
             [self.ifos[1], self.ifos[0]],
             [[0, -1], [-1, 0]]
         ):
+            if fixed_ifo not in ifos:
```

Comment:
It looks like 'ifos' needs defined in the docstring as well, particularly as the variable is not used anywhere else in this function